changelogs/fragments/2485-java_keystore-ssl_backend-parameter.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - java_keystore - added ``ssl_backend`` parameter for using the cryptography library instead of OpenSSL (https://github.com/ansible-collections/community.general/pull/2485).
```

Comment:
```suggestion
  - java_keystore - added ``ssl_backend`` parameter for using the cryptography library instead of the OpenSSL binary (https://github.com/ansible-collections/community.general/pull/2485).
```